raiden/tests/fixtures/raiden_network.py

```diff
@@ -19,7 +20,10 @@ def _raiden_cleanup(request, raiden_apps):
     """ Helper to do cleanup a Raiden App. """
     def _cleanup():
         for app in raiden_apps:
-            app.stop(leave_channels=False)
+            try:
+                app.stop(leave_channels=False)
+            except RaidenShuttingDown:
```

Comment:
I also had tried that. Reduced it a bit but did not really solve it.

Reply:
true, but it helps

Reply:
here is an example:

https://travis-ci.org/hackaugusto/raiden/jobs/356575198

with this change these errors don't occur

Reply:
okay let's see how it goes